control_panel_api/serializers.py

```diff
@@ -262,7 +262,7 @@ class Meta:
 
 
 class AppCustomerSerializer(serializers.Serializer):
-    email = serializers.EmailField()
```

Comment:
why not? `EmailSerializer(many=True)` or does that only work with ORM backed models?

Reply:
What is `EmailSerializer`? I can't find it in DRF. `EmailField` does not support `many=True` as an argument.

Reply:
I was guessing there would be one but `many=True` thing works for any serializer but this is a field so :+1: LGTM